lib/camunda-infra-stack.ts

```diff
@@ -36,52 +35,62 @@ export class CamundaInfraStack extends cdk.Stack {
     })
+    Tags.of(cluster).add("Name",`${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-cluster`)
 
     // Create SG for RDS
     const postgresSGAccess = new ec2.SecurityGroup(this, `rds-security-group-access`, {
       vpc: vpc,
-      allowAllOutbound: true,
-      description: 'SG to access Postgres'
+      allowAllOutbound: !!buildConfig.allowAllOutboundSGAccess,
+      description: 'SG to access Postgres',
     });
+    Tags.of(postgresSGAccess).add("Name",`${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-rds-sg-access`)
 
     const postgresSG = new ec2.SecurityGroup(this, `rds-security-group`, {
       vpc: vpc,
-      allowAllOutbound: true,
+      allowAllOutbound: !!buildConfig.allowAllOutboundSG,
       description: 'SG to attach to Postgres'
     });
     postgresSG.connections.allowFrom(postgresSGAccess, ec2.Port.tcp(5432), 'Ingress postgres from postgresSGAccess');
+    Tags.of(postgresSG).add("Name",`${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-rds-sg`)
 
     const secret = new rds.DatabaseSecret(this, 'Secret', {
-      username: buildConfig.DbUser
+      username: buildConfig.DbUser,
+      secretName: `${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-dbsecret`
     });
+    Tags.of(secret).add("Name",`${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-dbsecret`)
+
 
     // Create an RDS instance
     const rdsInstance = new rds.DatabaseInstance(this, 'CamundaInstance', {
       engine: rds.DatabaseInstanceEngine.postgres({version: rds.PostgresEngineVersion.VER_12_5}),
       instanceType: new ec2.InstanceType(`${buildConfig.DbInstClass}.${buildConfig.DbInstType}`),
       vpc: vpc,
-      vpcSubnets: {subnetType: ec2.SubnetType.PRIVATE},
+      vpcSubnets: {subnetType: ec2.SubnetType.ISOLATED},
       databaseName: buildConfig.DbName,
       credentials: rds.Credentials.fromSecret(secret),
-      securityGroups: [postgresSG]
+      securityGroups: [postgresSG],
+      instanceIdentifier: `${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-rds`
     })
+    Tags.of(rdsInstance).add("Name",`${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}-rds`)
 
     // Create a task definition
     const fargateTaskDefinition = new ecs.FargateTaskDefinition(this, 'camundaTaskDef', {
-      memoryLimitMiB: 512,
-      cpu: 256
+      memoryLimitMiB: +buildConfig.fargateMemoryLimitMiB,
+      cpu: +buildConfig.fargateCpu
     });
 
     // Add params to task definition
     fargateTaskDefinition.addContainer("camunda", {
       // Use an image from DockerHub
       image: ecs.ContainerImage.fromRegistry("camunda/camunda-bpm-platform"),
-      portMappings: [{ containerPort: 8080 }],
+      portMappings: [{ containerPort: +buildConfig.fargateContainerPort }],
       containerName: buildConfig.App,
       logging: ecs.LogDrivers.awsLogs({ streamPrefix: 'camunda' }),
```

Comment:
Here we need to replace the 'camunda' with `/aws/ecs/${buildConfig.Project}-${buildConfig.App}-${buildConfig.Environment}`